sdk/freestanding-execution-engine/README.md

```diff
@@ -0,0 +1,36 @@
+# Goal of the exercise
```

Comment:
I'd rephrase this as "Getting started with the Veracruz freestanding execution engine", or something like that, rather than "Goal of the exercise".